app/views/NewMessageView/index.tsx

```diff
@@ -93,6 +93,7 @@ const NewMessageView = () => {
 
 					return (
 						<UserItem
+							id={item._id}
```

Comment:
`_id`? Where does this come from?
It looks like it should be normalized to use `id` as everywhere else.